docs/01.CONFIGURATION.md

```diff
@@ -0,0 +1,249 @@
+  | Python: | `--js-parser=ON/OFF`                         |
+
+* ### Dump bytecode
+
+  This option can be used to display created bytecode in a human readable format.
+  This option is disabled by default.
+
+  | Options |                                              |
+  |---------|----------------------------------------------|
+  | C:      | `-DJERRY_PARSER_DUMP_BYTE_CODE=0/1`          |
+  | CMake:  | `-DJERRY_PARSER_DUMP_BYTE_CODE=ON/OFF`       |
+  | Python: | `--show-opcodes=ON/OFF`                      |
+
+* ### Dump RegExp bytecode
+
+  This option can be used to display created RegExp bytecode in a human readable format.
+  This option is disabled by default.
+
+  | Options |                                              |
+  |---------|----------------------------------------------|
+  | C:      | `-DJERRY_REGEXP_DUMP_BYTE_CODE=0/1`          |
+  | CMake:  | `-DJERRY_REGEXP_DUMP_BYTE_CODE=ON/OFF`       |
+  | Python: | `--show-regexp-opcodes=ON/OFF`               |
+
+* ### Strict RegExp
+
+  This option can be used to enable strict RegExp mode. The standard RegExp syntax is a lot stricter than what is common in current JavaScript implementations.
+  When enabled, this flag disables all of the non-standard, quality-of-life RegExp features in the engine. This option is disabled by default.
+
+  | Options |                                              |
+  |---------|----------------------------------------------|
+  | C:      | `-DJERRY_REGEXP_STRICT_MODE=0/1`             |
+  | CMake:  | `-DJERRY_REGEXP_STRICT_MODE=ON/OFF`          |
+  | Python: | `--regexp-strict-mode=ON/OFF`                |
+
+* ### Error messages
+
+  Enables error messages for thrown Error objects. By default, error messages are omitted to reduce memory usage.
+  Enabling this feature provides detailed error messages where available, like line information for Syntax errors, variable names for Reference errors, Type/Range error messages for built-in routines, etc.
+
+  | Options |                                              |
+  |---------|----------------------------------------------|
+  | C:      | `-DJERRY_ERROR_MESSAGES=0/1`                 |
+  | CMake:  | `--DJERRY_ERROR_MESSAGES=ON/OFF`             |
+  | Python: | `--error-messages=ON/OFF`                    |
+
+* ### Logging
+
+  This option can be used to enable log messages during runtime. When enabled the engine will use the `jerry_port_log` port API function to print relevant log messages.
+  This feature is disabled by default.
+
+  | Options |                                              |
+  |---------|----------------------------------------------|
+  | C:      | `-DJERRY_LOGGING=0/1`                        |
+  | CMake:  | `-DJERRY_LOGGING=ON/OFF`                     |
+  | Python: | `--logging=ON/OFF`                           |
+
+* ### LCache
+
+  This option enables the LCache, allowing faster access to object properties. See [Internals](04.INTERNALS.md) for further details.
```

Comment:
Does this link will work after webpage generation?

Reply:
Also should mention the memory consumption effect of this option.

Reply:
I tried it, it redirects to the correct page.